backend.py

```diff
@@ -2,41 +2,62 @@
 from flaskext.mysql import MySQL
 from datetime import datetime
 import logic
+import json
+from collections import defaultdict
 app = Flask(__name__)
 
-app.config['MYSQL_HOST'] = 'Osamas-MacBook-Pro.local'
-app.config['MYSQL_USER'] = 'Osama'
-app.config['MYSQL_PASSWORD'] = 'CSCI5030SLU2021'
+# app.config['MYSQL_HOST'] = 'Osamas-MacBook-Pro.local'
+# app.config['MYSQL_USER'] = 'Osama'
+# app.config['MYSQL_PASSWORD'] = 'CSCI5030SLU2021'
+# app.config['MYSQL_DATABASE_DB'] = 'wordsense'
+
+app.config['MYSQL_HOST'] = 'localhost'
+app.config['MYSQL_DATABASE_USER'] = 'pnkls'
+app.config['MYSQL_DATABASE_PASSWORD'] = 'root'
 app.config['MYSQL_DATABASE_DB'] = 'wordsense'
 
 mysql = MySQL(app)
 conn = mysql.connect()
 cursor = conn.cursor()
 
+def GetIndexing():
+    f = open("indexing.txt", "r")
+    dictionary = json.loads(f.read())
+    return dictionary
+
 @app.route('/', methods =["GET", "POST"])
 def homepage():
+    # Get the indexing in memory so that we have it until the application is closed
+    dictionary = GetIndexing()
+
     if request.method == "GET":
         sentence_List = ""
         error = ""
     if request.method == "POST":
         language_selected = request.form.get('language')
         Lang_ID = logic.SQLQuery(f"SELECT Lang_ID FROM Lang_Ref WHERE Lang_Desc = '{language_selected}';")
         part_of_speech_selected = request.form.get('partOfSpeech')
```

Comment:
I think we should add some comments about words like "Lang_ID, Lang_ref, and Land_desc so that a reviewer who doesn't know about the database can understand what these listings are.

Reply:
If we add comments to describe each variable that would turn out to be a lot more comments. And I think the variable names are self explanatory, aren't they ?

Reply:
I agree, however, I'm asking for some limited variables which are not easy to understand.